src/main/views/home.njk

```diff
@@ -1,9 +1,14 @@
-<html lang="en">
-  <head>
-    <title>Express application template</title>
-  </head>
-  <body>
-    <h1>Home page</h1>
-    <p>TODO: replace with a proper template in your application</p>
-  </body>
-</html>
+{% extends "govuk/template.njk" %}
+
+{% block head %}
```

Comment:
Would be good to create our own template which extends the gov uk one so that users don’t need to add the css and JavaScript in each view?